### DISCUSSION TITLE
Molten

### DISCUSSION THREAD

**Discussion by KevinPadillaCS** [AnonymousHacker1279/ImmersiveWeapons#91] [General]
Where do you find molten ore?

**Comment by kiamisha** [comment]
Hmmmm, I'm wondering too, I've searched the nether which I think would've had it, but there wasn't any.. where is the molten ore?

**Comment by AnonymousHacker1279** [comment]
Molten ore generates in the Nether between Y=0 and Y=48. It has a triangular generation shape, so higher concentrations are found in the middle of that: Y=24. There are usually 6-8 ores in a square placement for each vein.

They never generate next to air blocks so you'll need to dig for them or look under lava lakes. What I usually do is grab some fire resistance potions and just swim in the lava, because you'll often be able to see them under the surface.

Also, there is a [Wiki](https://anonymoushacker1279.tech/wiki/immersiveweapons/Molten_(Material)) page for Molten in case you need it for further reference.

**Comment by kiamisha** [comment]
Thank you, I am also wondering how do you make starstorm shards? I have the crystals, but I don't know what to do with them.

**Reply by AnonymousHacker1279** [reply]
Place a piston above a crystal and crush them, and it will drop shards. Unfortunately I don't have this documented yet on the wiki, but I am working on it.
